src/pages/AzurePreview.js

```diff
@@ -1,7 +1,7 @@
 import { useState } from 'react'
 import { div, h, h1, p } from 'react-hyperscript-helpers'
 import { ButtonOutline, ButtonPrimary, Link } from 'src/components/common'
-import { ReactComponent as AzureLogo } from 'src/images/azure-new.svg'
+import { ReactComponent as AzureLogo } from 'src/images/ms-azure.svg'
```

Comment:
Can you delete the previous icon you checked in? And did you optimize the SVG?

Reply:
Yes, and no -- will do